pyftdi/spi.py

```diff
@@ -384,6 +390,9 @@ def configure(self, url: Union[str, UsbDevice],
              frequency.
            * ``cs_count`` count of chip select signals dedicated to select
              SPI slave devices, starting from A*BUS3 pin
+           * ``cs_act_hi`` a bitfield specifying which SPI CS lines are active
```

Comment:
This does not conform to the other APIs so it is a bit confusing.
`b0` should match `CS0`, not SCLK.